src/commands/logout.js

```diff
@@ -1,16 +1,25 @@
 import auth from '../utils/auth';
 
-Cypress.Commands.add('logout', (returnTo) => {
-  const options = {
-    clientID: Cypress.env('auth0ClientId'),
-  };
+Cypress.Commands.add('logout', (options = {}) => {
+  const { returnTo, logoutUrl } = options;
 
-  if (returnTo) {
-    options.returnTo = returnTo;
-  }
+  const builtLogoutUrl = auth.client.buildLogoutUrl({
```

Comment:
I believe `buildLogoutUrl()` takes into account any settings the client has for a custom logout URL. So I'm curious whether there is a use case for `logoutUrl` being passed as a param? Meaning, when would someone pass in `logoutUrl` if the logout URL is automatically built in this method?

Reply:
Perhaps my question is related to the following... Could you explain why we are calling two different URLs in this method? Maybe I am mis-understanding the value returned by `buildLogoutUrl()`.

Reply:
Both questions are explained with the screenshot in [this comment](https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/44/files#r804924641). You can't open that URL from the Cypress if the base URL is `localhost` and the URL used before is the default URL added by nextjs-auth. And with the option to add a custom one one can make sure a custom one can be used too.

Reply:
I refactored it and removed `auth.client.buildLogoutUrl`: see https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/44/commits/4663ecd7cc1447dcc4a3416428e7290fcaaaa9f7. One for the reason mentioned in the [referenced comment](https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/44/files#r804924641) and two because it wasn't used before when the main maintainer of nextjs-auth0 [created a PR here](https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/14/files#diff-98adbd1748b77ce10a8d70a88b3c2cab8c57db2a8b7f305e3ec02ea14cc09a56).

Reply:
Resolved this conversation in favor of https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/44#discussion_r805134730